Lib/test/test_asyncio/test_tasks.py

```diff
@@ -2688,6 +2688,25 @@ def test_get_context(self):
         finally:
             loop.close()
 
+    def test_proper_refcounts(self):
```

Comment:
Can we have some link to the issue, for instance:

```suggestion
    def test_proper_refcounts(self):
    	# see: https://github.com/python/cpython/issues/126083
```

(I personally like using full links if I have the place since I can click on them in my IDE)